bnd.bnd

```diff
@@ -0,0 +1,16 @@
+Automatic-Module-Name: ${def;bsn}
+Bundle-Description: HiveMQ Java MQTT Client 
```

Comment:
We have all these values (name, license, scm) as variables in the gradle build script.
We could use them if we declare the bnd rules in the gradle build file: https://github.com/bndtools/bnd/blob/master/biz.aQute.bnd.gradle/README.md#instructing-bnd-on-how-to-build-your-bundle